_CodingChallenges/051.1-astar.md

```diff
@@ -32,6 +32,12 @@ contributions:
     source: "https://github.com/MkLahane/Game-Development"
     author:
       name: "Makarand Lahane"
+  - title: "Pathfinding A* in Windows Form using C#"
+    author:
+      name: "Trong Hieu Nguyen"
+      url:
```

Comment:
Sorry, but I totally overlooked this. If you don't want to specify a URL here, just leave this line out.